Builder/Exception/MissingAssociationAdmin.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\AdminBundle\Builder\Exception;
+
+/**
+ * Signals that a field description is missing an association admin when a form type requires it.
+ *
+ * @author ju1ius <ju1ius@laposte.net>
+ */
+final class MissingAssociationAdmin extends \LogicException
```

Comment:
Also, when using many exception, one of my colleagues recommends doing this: 

```
use My\Namespace\Exception;

throw new Exception\ThisShouldNotHappen();
throw new Exception\ThisShouldAlsoNotHappen();
throw new Exception\ThisShouldAbsolutelyNotHappen();
```

Related : http://mnapoli.fr/approaching-coding-style-rationally/